jupyterhub/values.yaml

```diff
@@ -618,7 +618,7 @@ prePuller:
     tolerations: []
     resources: {}
     serviceAccount:
-      create: true
+      create: false
```

Comment:
What's the reason for changing this to false?

Reply:
This was just to not create a service account by default, but I can change it to true again tomorrow (I'm on vacations and away from my PC until tomorrow)

Reply:
Just reverted to the old config

Reply:
If we're providing the ability to create a service account we should create one with the correct permissions.

Reply:
It turns out that the service account already had the correct permissions. However, due to the annotations present in that SA, it was getting deleted and throwing the error that the SA did not exist. I changed the SA and now everything works.

Feel free to make more suggestions, since my knowledge in k8s is still limited :)

Reply:
One test still failing, I will look into that

Reply:
Everything working, you can take a look again. Thanks for your time